dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_homepage_v2.feature

```diff
@@ -0,0 +1,29 @@
+@no_mobile
+Feature: Using the teacher homepage
+  @eyes
+  Scenario: Teacher can view sections on new teacher homepage
+    When I open my eyes to test "teacher homepage"
+    Given I create a teacher named "Hank"
+
+    # This line can be removed when the new teacher homepage is the default
+    Then I use a cookie to mock the DCDO key "teacher-homepage-v2" as "true"
+    And I sign in as "Hank" and go home
+    And I get levelbuilder access
+
+    # Create an empty section
+    And I create a new student section
+
+    # Create a section with a course
+    And I create a new student section assigned to "interactive-games-animations-2024" and save the section
+
+    # Create a student to join the second section
+    Given I create a student named "Bobby"
+    Then I sign in as "Bobby"
+    And I join the section
+
+    # Navigate to the new teacher homepage
+    Then I sign out and sign in as "Hank"
```

Comment:
Optional nit: Other places that I've seen a create a teacher they have been named "Teacher <name>". We could use Teacher Hank here to be more consistent